Cannot compile against PG11

On CentOS 7 64 bit, PostgreSQL 11.1 I cannot compile version 2.0.0-alpha.2:

```
make
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -fPIC -I./include/ -fvisibility=hidden -I. -I./ -I/usr/pgsql-11/include/server -I/usr/pgsql-11/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include  -c -o src/tds_fdw.o src/tds_fdw.c
src/tds_fdw.c: In function ‘tdsGetColumnMetadata’:
src/tds_fdw.c:1395:72: error: invalid type argument of ‘->’ (have ‘FormData_pg_attribute’)
       column->attr_oid = festate->attinmeta->tupdesc->attrs[local_ncol]->atttypid;
                                                                        ^
src/tds_fdw.c:1404:65: error: invalid type argument of ‘->’ (have ‘FormData_pg_attribute’)
      local_name = festate->attinmeta->tupdesc->attrs[local_ncol]->attname.data;
                                                                 ^
src/tds_fdw.c:1417:72: error: invalid type argument of ‘->’ (have ‘FormData_pg_attribute’)
       column->attr_oid = festate->attinmeta->tupdesc->attrs[local_ncol]->atttypid;
                                                                        ^
src/tds_fdw.c:1443:63: error: invalid type argument of ‘->’ (have ‘FormData_pg_attribute’)
    column->attr_oid = festate->attinmeta->tupdesc->attrs[ncol]->atttypid;
                                                               ^
In file included from /usr/pgsql-11/include/server/postgres.h:47:0,
                 from src/tds_fdw.c:32:
src/tds_fdw.c:1461:47: error: invalid type argument of ‘->’ (have ‘FormData_pg_attribute’)
       festate->attinmeta->tupdesc->attrs[ncol]->attname.data)
                                               ^
/usr/pgsql-11/include/server/utils/elog.h:107:14: note: in definition of macro ‘ereport_domain’
    errfinish rest; \
              ^
src/tds_fdw.c:1457:5: note: in expansion of macro ‘ereport’
     ereport(WARNING,
     ^
make: *** [src/tds_fdw.o] Error 1

```

I've pulled the commit 3719a995b0ae3fc4c4b390dd8a2820d54b88e18a from the repository, and it seems a little better but I've something wrong with clang or llvm:

```
$ make
/opt/rh/llvm-toolset-7/root/usr/bin/clang -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -I./include/ -fvisibility=hidden -I. -I./ -I/usr/pgsql-11/include/server -I/usr/pgsql-11/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o src/tds_fdw.bc src/tds_fdw.c
make: /opt/rh/llvm-toolset-7/root/usr/bin/clang: Command not found
make: *** [src/tds_fdw.bc] Error 127
```

I cannot get llvm-toolset from the yum repository, so cannot try. Suggestions?

Even with `llvm-toolset-7` it is still possible to compile with:
```
make USE_PGXS=1 CLANG=true install
```
As instructed at https://github.com/powa-team/pg_stat_kcache/issues/15#issuecomment-422691873

However the install breaks, with our without `-k` parameter:
```
[root@ea24bc4f805d tds_fdw]# PATH=/usr/pgsql-11/bin:$PATH make -k USE_PGXS=1 CLANG=true install
true -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -I./include/ -fvisibility=hidden -I. -I./ -I/usr/pgsql-11/include/server -I/usr/pgsql-11/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o src/tds_fdw.bc src/tds_fdw.c
true -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -I./include/ -fvisibility=hidden -I. -I./ -I/usr/pgsql-11/include/server -I/usr/pgsql-11/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o src/options.bc src/options.c
true -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -I./include/ -fvisibility=hidden -I. -I./ -I/usr/pgsql-11/include/server -I/usr/pgsql-11/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o src/deparse.bc src/deparse.c
/usr/bin/mkdir -p '/usr/pgsql-11/lib'
/usr/bin/mkdir -p '/usr/pgsql-11/share/extension'
/usr/bin/mkdir -p '/usr/pgsql-11/share/extension'
/usr/bin/mkdir -p '/usr/pgsql-11/doc/extension'
/usr/bin/install -c -m 755  tds_fdw.so '/usr/pgsql-11/lib/tds_fdw.so'
/usr/bin/install -c -m 644 .//tds_fdw.control '/usr/pgsql-11/share/extension/'
/usr/bin/install -c -m 644 .//sql/tds_fdw--2.0.0-alpha.2.sql  '/usr/pgsql-11/share/extension/'
/usr/bin/install -c -m 644 .//README.tds_fdw.md '/usr/pgsql-11/doc/extension/'
/usr/bin/mkdir -p '/usr/pgsql-11/lib/bitcode/tds_fdw'
/usr/bin/mkdir -p '/usr/pgsql-11/lib/bitcode'/tds_fdw/src/
/usr/bin/install -c -m 644 src/tds_fdw.bc '/usr/pgsql-11/lib/bitcode'/tds_fdw/src/
/usr/bin/install: cannot stat 'src/tds_fdw.bc': No such file or directory
make: *** [install] Error 1
```
```
[root@ea24bc4f805d tds_fdw]# PATH=/usr/pgsql-11/bin:$PATH make USE_PGXS=1 CLANG=true install
true -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -I./include/ -fvisibility=hidden -I. -I./ -I/usr/pgsql-11/include/server -I/usr/pgsql-11/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o src/tds_fdw.bc src/tds_fdw.c
true -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -I./include/ -fvisibility=hidden -I. -I./ -I/usr/pgsql-11/include/server -I/usr/pgsql-11/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o src/options.bc src/options.c
true -Wno-ignored-attributes -fno-strict-aliasing -fwrapv -O2  -I./include/ -fvisibility=hidden -I. -I./ -I/usr/pgsql-11/include/server -I/usr/pgsql-11/include/internal  -D_GNU_SOURCE -I/usr/include/libxml2  -I/usr/include -flto=thin -emit-llvm -c -o src/deparse.bc src/deparse.c
/usr/bin/mkdir -p '/usr/pgsql-11/lib'
/usr/bin/mkdir -p '/usr/pgsql-11/share/extension'
/usr/bin/mkdir -p '/usr/pgsql-11/share/extension'
/usr/bin/mkdir -p '/usr/pgsql-11/doc/extension'
/usr/bin/install -c -m 755  tds_fdw.so '/usr/pgsql-11/lib/tds_fdw.so'
/usr/bin/install -c -m 644 .//tds_fdw.control '/usr/pgsql-11/share/extension/'
/usr/bin/install -c -m 644 .//sql/tds_fdw--2.0.0-alpha.2.sql  '/usr/pgsql-11/share/extension/'
/usr/bin/install -c -m 644 .//README.tds_fdw.md '/usr/pgsql-11/doc/extension/'
/usr/bin/mkdir -p '/usr/pgsql-11/lib/bitcode/tds_fdw'
/usr/bin/mkdir -p '/usr/pgsql-11/lib/bitcode'/tds_fdw/src/
/usr/bin/install -c -m 644 src/tds_fdw.bc '/usr/pgsql-11/lib/bitcode'/tds_fdw/src/
/usr/bin/install: cannot stat 'src/tds_fdw.bc': No such file or directory
make: *** [install] Error 1
```
I guess we probablyt need to change something at the source code, but I am not sure what exactly.

@GeoffMontee any idea?

--- a/InstallCentOS.md
+++ b/InstallCentOS.md
@@ -35,6 +35,19 @@ sudo rpm -ivh pgdg-centos95-9.5-2.noarch.rpm
 sudo yum install postgresql95 postgresql95-server postgresql95-libs postgresql95-devel
 ```
 
+### CentOS7 and PostgreSQL >= 11
+
+When using the official PostgreSQL packages from postgresql.org, JIT with bitcode is enabled by default and will require llvm5 and `clang` from LLVM7 installed at `/opt/rh/llvm-toolset-7/root/usr/bin/clang`
+
+You have LLVM5 at the standard CentOS7 repositories, but not LLVM7, so you will need install the CentOS Software collections.
+
+You can easily do it with the following commands:
+
+```bash
+sudo yum install centos-release-scl
+sudo yum install llvm-toolset-7-clang llvm5.0
+```
+
 ### Install tds_fdw
 
 #### Build from release package

